saleor/product/models.py

```diff
@@ -92,6 +92,7 @@ class Product(models.Model):
         ProductType, related_name='products', on_delete=models.CASCADE)
     name = models.CharField(max_length=128)
     description = models.TextField()
+    seo_description = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
I'd set max_length to 300, as that's the maximum displayed by Google and Facebook